### ISSUE TITLE
Game Day not functioning

### ISSUE THREAD

**Issue by clapdog** [mtgred/netrunner#1171]
Chrome player - Clicking Game Day is nonresponsive. Second attempt triggers card cost but it doesn't draw any cards.  Every time I've tried it in concert with Beach Party, so it should draw up to 10 cards.  Game Day also doesn't appear in heap after it's played.

Apologies if this issue is poorly reported - I'm new to Github and have no coding knowledge...

**Comment by JoelCFC25** [comment]
Thanks for the report, I've confirmed the cause. @Saintis: the `hand-size` function is receiving the wrong number of arguments. Changing it to `(hand-size state :runner)` fixes it.

**Comment by clapdog** [comment]
Thanks for the reply and attention to the issue - I don't speak code at all or understand github sorry -  but the problem still seems to be happening...

**Comment by JoelCFC25** [comment]
When someone submits a pull request to fix a particular bug, there are two more things that have to happen before you see it working correctly--it first has to get merged into the master branch, and then the server needs to be restarted to put all merged changes into production. Neither of these steps has happened yet, but it occurs approximately once a week. Keep an eye on the News header in the main chat for an indication of a restart and a summary of new/fixed items.

**Comment by clapdog** [comment]
thanks so much for walking me through it :)
